src/odoc/bin/main.ml

```diff
@@ -511,13 +527,28 @@ module Indexing = struct
         value & opt_all convert_fpath []
         & info ~doc ~docv:"FILE" [ "file-list" ])
     in
+    let include_rec =
```

Comment:
What's the point of this option ? The `inputs` term could also accept directories, which removes an exception case and is common among CLI tools.

Reply:
In any case, this option will change to `-P` and `-L` soon: we will use the index for sidebars too, and sidebars require those options.